src/components/atoms/DatasetInfoBox.tsx

```diff
@@ -32,6 +62,15 @@ const DatasetInfoBox: React.FC<DatasetInfoBoxProps> = ({ title, description, tag
                     </Badge>
                 ))}
             </Pane>
+            <Pane marginTop="auto" display="flex" flexDirection="column">
+                <Text size={300}>
+                    URL:{' '}
+                    <Link href={url} size={300}>
+                        {url}
+                    </Link>
```

Comment:
This still doesn't work with a single click.
Tested on Firefox.

Reply:
Does work with a "Right click -> Open in new tab".

I suspect someone put an `onClick` handler on the cards which is interfering with everything else that wants to be clicked on inside it.

You might need to 

a) change the onClick on the card
b) do some trickery with event propagation and bubbling

Reply:
Doesn't work in Safari either.

Reply:
I'd appreciate it if you could test this properly before re-requesting a review.

Reply:
It worked on my firefox. I tested it before I pushed. I will look more into the problem

Reply:
By the way, I am referring to the data explorer links. Did those work on your firefox as well?

Reply:
My bad, I will fix it.